manta-js/examples/asset-webpack-ts/index.ts

```diff
@@ -124,9 +124,12 @@ const toPrivateOnlySignTest = async () => {
     const initalPrivateBalance = await privateWallet.getPrivateBalance(assetId);
     console.log("The inital private balance is: ", initalPrivateBalance.toString());
 
-    const signResult = await privateWallet.toPrivateSend(assetId, amount, polkadotConfig.polkadotSigner, polkadotConfig.polkadotAddress);
+    const signResult = await privateWallet.toPrivateBuild(assetId, amount, polkadotConfig.polkadotSigner, polkadotConfig.polkadotAddress);
```

Comment:
whats the difference in these fns?

Reply:
The send tries to submit them and this is supposed to be sign only.